zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java

```diff
@@ -416,14 +417,18 @@ private void copyRunFileToContainer(String containerId)
       intpKeytab = properties.getProperty("submarine.hadoop.keytab", "");
     }
     if (StringUtils.isBlank(intpKeytab)) {
-      // 3.4) jdbc interpreter properties keytab file
+      // 3.4) livy interpreter properties keytab file
+      intpKeytab = properties.getProperty("zeppelin.livy.keytab", "");
+    }
+    if (StringUtils.isBlank(intpKeytab)) {
+      // 3.5) jdbc interpreter properties keytab file
```

Comment:
This seems work, but it is weird to see interpreter specific thing in `DockerInterpreterProcess`. We might need to think about other approach for it. @liuxunorg

Reply:
Because the key table property names set by each interpreter are different, in `DockerInterpreterProcess.java`, the keytab file is now copied to the container in this way.
Let the interpreter in the container also be authenticated properly.

Reply:
so is there anything I can improve?

Reply:
I am fine with current implementation also not so elegant. Maybe in future we should consider about a general property for interpreter keytab, e.g. `zeppelin.interpreter.keytab`. But it is beyond the scope of this PR, and could be done in another ticket.

Reply:
Could you add a TODO comment so that we won't miss this issue ?

Reply:
Yes, I added todo.

Reply:
what @zjffdu said.